cfgov/v1/wagtail_hooks.py

```diff
@@ -30,7 +39,8 @@ def share_the_page(request, page):
 
     share(page, is_sharing, is_live)
     configure_page_revision(page, is_sharing, is_live)
-    flush_akamai(page, is_live)
+    if is_live:
+        flush_akamai(page)
```

Comment:
@chosak If we attempt a flush, it does get logged (even if it fails). This would return `False` if we shouldn't be flushing at all, but it seemed a little out of scope to add logging to parts of the code I didn't change.